airgun/entities/rhai/overview.py

```diff
@@ -0,0 +1,24 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.rhai import OverviewDetailsView
+
+
+class OverviewEntity(BaseEntity):
+
+    @property
```

Comment:
do simple def read()
view = self.navigate_to(self, "Details")
return view.read()

and then in test just assert necessary values as a strings or transform then in int on the fly if it is absolutely necessary